__tests__/app.test.js

```diff
@@ -49,4 +48,54 @@ describe('APP be-nc-news', () =>
             })
         })
     })
+    describe('GET /api/articles', () =>
+    {
+        test('GET: will return a status of 200', () => 
+        {
+            return request(app).get('/api/articles').expect(200)
```

Comment:
its kind of redundant to have a test for this when you are also testing for this in the next tests :)